docs/products/opensearch/howto/sample-dataset.rst

```diff
@@ -95,14 +95,14 @@ OpenSearch Python client offers a helper called bulk() which allows us to send m
 
 Get data mapping with Python
 ----------------------------
-When we sent the data to OpenSearch cluster on :ref:`load the data with Python <load-data-with-python>`, no data structure was specified.
-This is possible because OpenSearch uses **dynamic mapping** to automatically add the fields to their type mapping. 
+When we sent the data to OpenSearch cluster on :ref:`load the data with Python <load-data-with-python>`, no data structure was specified. 
+As default, OpenSearch uses **dynamic mapping** to automatically add the fields to their type mapping. 
```

Comment:
I feel that "as default" is not exactly right. It is when the schema is not provided it uses dynamic mapping.

Reply:
can you see if it makes sense now?